Volume/Models/Organization.swift

```diff
@@ -23,11 +23,17 @@ struct Organization: Hashable, Identifiable {
     let websiteUrl: URL?
     
     init(from organization: OrganizationFields) {
-        self.backgroundImageUrl = URL(string: organization.backgroundImageUrl)
+        self.backgroundImageUrl = {
+            guard let stringUrl = organization.backgroundImageUrl else { return nil }
+            return URL(string: stringUrl)
+        }()
         self.bio = organization.bio
         self.categorySlug = OrganizationType(rawValue: organization.categorySlug) ?? .academic
```

Comment:
Unrelated to your changes but would we be able to create a default `.none` value for the organization category? Ideally the tag would just not be there if the organization does not have a specified category.

Reply:
Yea you're right, but when this was made (and even atm) all organizations have a category. For now, we can ignore but definitely something we should think about when designing the new Flyers/Organizations stuff.